test/functional/tests/stress/test_stress_io_small_device.py

```diff
@@ -0,0 +1,107 @@
+#
+# Copyright(c) 2019 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+#
+
+import pytest
+from datetime import timedelta
+
+from api.cas import casadm, casadm_parser
+from api.cas.cache_config import CacheMode, CacheLineSize, CleaningPolicy
+from storage_devices.disk import DiskType, DiskTypeSet, DiskTypeLowerThan
+from core.test_run import TestRun
+from test_tools.fio.fio import Fio
+from test_tools.fio.fio_param import ReadWrite, IoEngine
+from test_utils.size import Size, Unit
+
+stress_time = timedelta(minutes=30)
+
+
+@pytest.mark.parametrize("cache_mode", [CacheMode.WT, CacheMode.WA, CacheMode.PT])
+@pytest.mark.parametrize("core_number", [1, 4])
+@pytest.mark.parametrize("cache_line_size", CacheLineSize)
+@pytest.mark.require_disk("cache", DiskTypeSet([DiskType.optane, DiskType.nand]))
+@pytest.mark.require_disk("core", DiskTypeLowerThan("cache"))
+def test_stress_small_cas_device(cache_mode, cache_line_size, core_number, cleaning_policy=None):
+    """
+        title: Stress test for verifying data on small CAS devices.
+        description: |
+          Validate the ability of the CAS to handle many iops when device is small
+          using different cache modes, cache line sizes and core numbers.
+        pass_criteria:
+          - No system crash.
+          - Md5 sums of core device and cache device are equal.
```

Comment:
you are checking core and core device (cache shouldn't have md5 sum equal to core